<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBook</class>
 <widget class="QWidget" name="FormBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить автора</string>
  </property>
  <widget class="QPushButton" name="pushBook">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>327</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>id Автора</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество страниц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSize">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Издательство)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePublic">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Год издания</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineYear">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
